homeassistant/components/goe_charger/common.py

```diff
@@ -0,0 +1,144 @@
+"""Common code go-e Charger integration."""
```

Comment:
This file should be moved to Python library hosted on pypi.
First bullet point: https://developers.home-assistant.io/docs/development_checklist/

Reply:
Or use the already existing library: https://github.com/cathiele/goecharger